### DISCUSSION TITLE
How to set up grid points for the write-component grid?

### DISCUSSION THREAD

**Discussion by XiaSun-Atmos** [ufs-community/ufs-srweather-app#336] [Q&A]
Hi, I am trying to set up a user-defined region. I understand the grid points for x and y direction for the write-component grid are different from the number of cells from the ESGgrid. How should `WRTCMP_nx` and `WRTCMP_ny `be set up? Thanks.

**Comment by ShawnCebulaNOAA** [comment]
Hello @XiaSun-Atmos!

Sorry for the delay in getting back to you!  Were you able to get this answered?  If not, I can check with the NOAA team to see if they're able to help!

Shawn Cebula - EPIC End User Support

**Comment by XiaSun-Atmos** [comment]
Thanks @ShawnCebulaNOAA for checking. Gerard from DTC has helped me with this issue offline.
Xia
